code_review.md

```diff
@@ -0,0 +1,34 @@
+Code Review Guidelines
+======================
+
+(some of the steps below don't necessarily need to be done in the order they are presented)
+
+1. Assign yourself to the pull request
+
+1. Verify that all of the GitHub checks have passed (Travis, etc.)
+
+1. Check out the branch in your local environment
+
+1. If you're reviewing code in an engine, regenerate the test app
+
+1. Verify that all the tests pass in your local environment
+
+1. Run the code in your local environment
```

Comment:
> Run the code in your local environment

Clarify to include?: "e.g., test the feature in your browsers or however else appropriate"